### DISCUSSION TITLE
How to set a precision for an expression unit separately?

### DISCUSSION THREAD

**Discussion by aweirdocc** [handsontable/hyperformula#1449] [API Questions]
I have such a requirement as the title says.
I found the `precisionRounding` configuration in the document, but this is a global configuration.
I wonder if there are any other good suggestions and methods, thank you!

**Comment by sequba** [comment]
Hi @aweirdocc, thanks for reaching out to us.

`precisionRounding` is indeed a global configuration option. HyperFormula does not support setting it per cell, but your UI could control how many decimal digits are displayed for each cell.

Can you describe your application and how it uses HyperFormula? Once I understand your use case better, I could propose a suitable solution.

**Reply by aweirdocc** [reply]
Thanks for your reply. You are right. I think I can handle this part of the logic in the custom renderer. I tried it and it works.
